packages/gatsby-theme-carbon/src/components/Homepage/homepage.scss

```diff
@@ -73,6 +73,20 @@
   }
 }
 
+.container--header--nav .#{$prefix}--grid {
+  margin: 0;
+
+  @include carbon--breakpoint("lg") {
+    padding-left: calc(
+      2% + 2rem
+    ); //2rem to account for 2rem left padding default on grid
+  }
+
+  @include carbon--breakpoint("max") {
+    padding-left: 2%;
```

Comment:
Hmmm this is a little weird, where'd 2% come from?

Reply:
@vincentsnagg when I left it at. 2rem there was no buffer on the left and it just looked weird. Let me know your thoughts on how to best handle this?

Reply:
I was following the margin styling on the design site as close as I could:
https://www.ibm.com/design/